grafana/dashboards/default_dashboard.json

```diff
@@ -563,7 +488,7 @@
         "y": 9
       },
       "hiddenSeries": false,
-      "id": 8,
+      "id": 11,§
```

Comment:
```suggestion
      "id": 11,
```
That's probably the `Â` char that was causing some errors

Reply:
yes, thats it. Thanks!